packages/target-decisioning-engine/src/contextProvider.js

```diff
@@ -88,6 +88,24 @@ export function createMboxContext(mboxRequest) {
   };
 }
 
+/**
+ * @param { import("@adobe/target-tools/delivery-api-client/models/Context").Context } context
+ * @return { import("../types/DecisioningContext").GeoContext }
+ */
+export function createGeoContext(geoContext) {
+  if (!geoContext || geoContext === {}) {
+    // eslint-disable-next-line no-param-reassign
+    geoContext = "";
+  }
```

Comment:
I'm not sure i understand why you're setting `geoContext` to an empty string.  That would result in an error below when you look up geoContext.latitude, etc.  You could just default to empty object instead...
```
export function createGeoContext(geoContext={}) {
```

Reply:
I was trying to build a geo context object similar to the PageContext object with empty values for the keys. Now defaulting to an empty object.